src/core/server/rendering/views/template.tsx

```diff
@@ -0,0 +1,134 @@
+        <link
+          rel="apple-touch-icon"
+          sizes="180x180"
+          href={`${uiPublicUrl}/favicons/apple-touch-icon.png`}
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="32x32"
+          href={`${uiPublicUrl}/favicons/favicon-32x32.png`}
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="16x16"
+          href={`${uiPublicUrl}/favicons/favicon-16x16.png`}
+        />
+        <link rel="manifest" href={`${uiPublicUrl}/favicons/manifest.json`} />
+        <link
+          rel="mask-icon"
+          color="#e8488b"
+          href={`${uiPublicUrl}/favicons/safari-pinned-tab.svg`}
+        />
+        <link rel="shortcut icon" href={`${uiPublicUrl}/favicons/favicon.ico`} />
+        <meta name="msapplication-config" content={`${uiPublicUrl}/favicons/browserconfig.xml`} />
+        <meta name="theme-color" content="#ffffff" />
+        <Styles darkMode={darkMode} />
+      </head>
+      <body>
+        {createElement('kbn-csp', {
+          data: JSON.stringify({ strictCsp }),
+        })}
+        {createElement('kbn-injected-metadata', { data: JSON.stringify(injectedMetadata) })}
+        <div
+          className="kibanaWelcomeView"
+          id="kbn_loading_message"
+          style={{ display: 'none' }}
+          data-test-subj="kbnLoadingMessage"
+        >
+          <div className="kibanaLoaderWrap">
+            <div className="kibanaLoader" />
+            <div className="kibanaWelcomeLogoCircle">
+              <div className="kibanaWelcomeLogo" />
+            </div>
+          </div>
+        </div>
+
+        <div
+          className="kibanaWelcomeView"
+          id="kbn_legacy_browser_error"
+          style={{ display: 'none' }}
+        >
+          <div className="kibanaLoaderWrap">
+            <div className="kibanaWelcomeLogoCircle">
+              <div className="kibanaWelcomeLogo" />
+            </div>
+          </div>
+
+          <h2 className="kibanaWelcomeTitle">
+            {i18n('common.ui.legacyBrowserTitle', {
+              defaultMessage: 'Please upgrade your browser',
+            })}
+          </h2>
+          <div className="kibanaWelcomeText">
+            {i18n('common.ui.legacyBrowserMessage', {
+              defaultMessage:
+                'This Kibana installation has strict security requirements enabled that your current browser does not meet.',
+            })}
+          </div>
+        </div>
+
+        <script>
+          {`
+            // Since this is an unsafe inline script, this code will not run
+            // in browsers that support content security policy(CSP). This is
+            // intentional as we check for the existence of __kbnCspNotEnforced__ in
+            // bootstrap.
+            window.__kbnCspNotEnforced__ = true;
+          `}
+        </script>
+        <script src={bootstrapScriptUrl} />
```

Comment:
The bootstrap.js endpoint needs to be migrated (different PR is fine with me). Can we make sure this is captured in an issue?